data/selectors.js

```diff
@@ -19,6 +19,7 @@ const elements = {
   "spanDown": ".ant-input-number-handler-down",
   "ageError": ".ant-form-item-explain-error",
   "imageSquare": ".ant-upload-select-picture-card",
+  "ageRequired": ".ant-form-item-explain-error",
```

Comment:
Maybe we could combine two selectors into one? I already have that selector on line 17 but I want to do some refactoring, so it will be `.ant-form-item-explain-error`.
```suggestion
  "error": ".ant-form-item-explain-error",
```

Reply:
Delete your code from 22 lines and use this.
![image](https://user-images.githubusercontent.com/80077934/113489351-93b48600-9478-11eb-9d75-6f2cdb685f15.png)

Reply:
Done